Rakefile

```diff
@@ -1,7 +1,7 @@
 require "logstash/devutils/rake"
 
 desc "Get jars"
-task :get_jars do
+task :vendor do
```

Comment:
For context: https://github.com/elastic/logstash-devutils/blob/master/lib/logstash/devutils/rake/vendor.rake